### DISCUSSION TITLE
ILS freq automatic?

### DISCUSSION THREAD

**Discussion by MStallen** [flybywiresim/aircraft#4918] [Help] [answered]
In previous versions of the DEV-mod, when I selected an ILS-approach in the FMC, the corrsponding ILS freq was inserted automatically. Now I've to put in the freq. manually. Do I do something wrong. (for instance ILS04R on LFMN)

**Comment by sidnov** [comment, marked as answer]
Are you using any airport add-ons? Automatic ILS frequency in RAD NAV works with default airports and the addons which have them configured properly

**Reply by tracernz** [reply]
Nah, it’s bugged even then. I just found a new way to reliably tune it by keeping a bit more data from the coherent LOAD_AIRPORT call this evening, but ran out of time for a PR. It seems to work on the airports where the default method never worked as well.
